_posts/2019-14-11-designup-bangalore.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: event
+title:  "Open Design @ DesignUP!: Bangalore, Nov 14th"
+date:   2019-14-11
```

Comment:
I think the same applies here about the date format YYYY-MM-DD as @jancborchardt mentioned in the other talk PR.

Reply:
Ah ok, I'm not actually sure how to change this and re-submit but we might want some documentation/guidance that the date format is that way rather than the way I accidentally did it :(